substrate/frame/support/procedural/Cargo.toml

```diff
@@ -32,14 +33,42 @@ expander = { workspace = true }
 sp-crypto-hashing = { workspace = true }
 
 [dev-dependencies]
+codec = { features = [
+	"derive",
+	"max-encoded-len",
+], workspace = true }
 regex = { workspace = true }
+sp-metadata-ir = { workspace = true }
+scale-info = { features = ["derive"], workspace = true }
+sp-core = { workspace = true }
+sp-io = { workspace = true }
+sp-runtime = { features = [
+	"serde",
+], workspace = true }
+frame-system = { workspace = true, default-features = true }
+frame-support = { workspace = true }
+pretty_assertions = { workspace = true }
+static_assertions = { workspace = true, default-features = true }
 
 [features]
 default = ["std"]
-std = ["sp-crypto-hashing/std"]
+std = [
+	"codec/std",
+	"frame-support/std",
+	"frame-system/std",
+	"scale-info/std",
+	"sp-core/std",
+	"sp-crypto-hashing/std",
+	"sp-io/std",
+	"sp-metadata-ir/std",
+	"sp-runtime/std",
+]
```

Comment:
`frame-system` is imported with `default-features`, so I guess this is redundant?
why not importing all dev dependencies with default feature, or otherwise not set `frame-system` with `default-features`, nor `static_assertions`.